### PR TITLE
Stop compiling old oopMapStubGenerator

### PR DESCRIPTION
**Pull request by shipilev**
This code is currently not used, and it might not be used in future too. There is no way to test if it works, currently. It is the major source of build failure when compiling x86_32. So, it would be better to `#if 0` it, or even remove it wholesale.

Additional testing:
 - [x] Linux x86_64 `tier1_loom`

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/loom pull/66/head:pull/66` \
`$ git checkout pull/66`

Update a local copy of the PR: \
`$ git checkout pull/66` \
`$ git pull https://git.openjdk.java.net/loom pull/66/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 66`

View PR using the GUI difftool: \
`$ git pr show -t 66`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/loom/pull/66.diff">https://git.openjdk.java.net/loom/pull/66.diff</a>

</details>

### CODE REVIEW

**Comment by pron** [comment]
Thank you! You can `/integrate`.

File: src/hotspot/share/compiler/oopMapStubGenerator.hpp [hunk 1]
```diff
@@ -51,3 +54,7 @@ class OopMapStubGenerator {
 };
 
 #endif
+
+#else
+#error This code is not used
+#endif
```
Comment (AlanBateman) [inline src/hotspot/share/compiler/oopMapStubGenerator.hpp:60]: Can you check Windows, I suspect it might have to be excluded.

Comment (shipilev) [reply src/hotspot/share/compiler/oopMapStubGenerator.hpp:60]: Um, Windows builds fail here:
 https://github.com/shipilev/loom/runs/3716995428?check_suite_focus=true

Which implies `oopMapStubGenerator_x86.cpp` is still compiled? What. Let me see...

Comment (shipilev) [reply src/hotspot/share/compiler/oopMapStubGenerator.hpp:60]: See #68.

**Pull request closed without merging**